src/window.rs

```diff
@@ -46,6 +46,8 @@ pub enum Key {
 pub enum Event {
     Resize { width: u32, height: u32 },
     Keyboard { key: Key, pressed: bool },
+    Pointer { position: [f32;2] },
```

Comment:
we should probably use `mint::Vector2D<f32>` instead for consistency with the rest of the API